connectors/opa/opa_connector.go

```diff
@@ -239,20 +228,36 @@ func buildNewEnfrocementAction(transformAction interface{}) (*pb.EnforcementActi
 			log.Println("Warning: unknown format of used policy information. Skipping policy", action)
 		}
 
-		if result, ok := action["result"].(string); ok {
+		if result, ok := action["action_name"].(string); ok {
 			switch result {
```

Comment:
Governance Actions should be implemented as a taxonomy rather than hard coded.  I believe I made this comment to the original PR.  Not sure if there was some reason it was deferred.  If so, please share.

Reply:
@simanadler Actions are given in taxonomy form in [action_struct.rego file](https://github.com/rohithdv/the-mesh-for-data/blob/opa-restructure-ety-changes/third_party/opa/data-and-policies/data_policies/action_struct.rego). Is this what you are referring to ? 
in opa_connector.go we are reusing the same action names provided in the action_struct.rego file above as there it is difficult (to my  understanding) to refer to rego files from go files. Please let me know if this is ok. Thanks.